infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -328,16 +328,22 @@ public void setReplicationRemainingEntries(String clusterId, long remainingEntri
         TxBuilder txBuilder = corfuStore.tx(NAMESPACE);
         txBuilder.update(REPLICATION_STATUS_TABLE, key, val, null);
         txBuilder.commit();
+
+        log.trace("setReplicationRemainingEntries: clusterId: {}, remainingEntries: {}, type: {}",
+                clusterId, remainingEntries, type);
     }
 
     public Map<String, ReplicationStatusVal> getReplicationRemainingEntries() {
         Map<String, ReplicationStatusVal> replicationStatusMap = new HashMap<>();
         QueryResult<CorfuStoreEntry<ReplicationStatusKey, ReplicationStatusVal, ReplicationStatusVal>> entries =
-                corfuStore.query(NAMESPACE).executeQuery(REPLICATION_STATUS_TABLE, (x) -> {return true;});
+                corfuStore.query(NAMESPACE).executeQuery(REPLICATION_STATUS_TABLE, record -> true);
 
         for(CorfuStoreEntry<ReplicationStatusKey, ReplicationStatusVal, ReplicationStatusVal>entry : entries.getResult()) {
             replicationStatusMap.put(entry.getKey().getClusterId(), entry.getPayload());
         }
+
+        log.debug("getReplicationRemainingEntries: replicationStatusMap size: {}", replicationStatusMap.size());
```

Comment:
do we also need to print the number of remainingEntries?

Reply:
I think number of remainingEntries is fine, because the consumer will only check it is 0 or not.